src/features/content-explorer/content-explorer-modal-container/ContentExplorerModalContainer.js

```diff
@@ -8,6 +8,8 @@ import NewFolderModal from '../new-folder-modal';
 
 class ContentExplorerModalContainer extends Component {
     static propTypes = {
+        /** Breadcrumb component options */
+        breadcrumbProps: PropTypes.object,
```

Comment:
Can we create a type for this? Or perhaps define the prop types in the object instead of just `object`?